old/selector/__init__.py

```diff
@@ -2,19 +2,19 @@
 
 import astropy.units.quantity
 
-from old_collector import Collector
-import old_collector.vskyutil as vs
-import old_collector.sb as sb
+from collector import Collector
+import collector.vskyutil as vs
+import collector.sb as sb
```

Comment:
Not sure if this will work? I mean, it doesn't matter if we can't run the old selector at this point. I'm just curious if this is going to try to run something in this collector or in the new one?

Reply:
You are right. Probably following the namespace should be `old.collector`